projects/components/src/lib/directives/responsive-input/responsive-input.directive.ts

```diff
@@ -21,10 +25,29 @@ export class ResponsiveInputDirective implements AfterViewChecked {
 
     private breakPoint = 768;
 
+    /**
+     * The directive will be a no-op if it's disabled. It does not support being changed after rendering.
+     */
+    @Input('vcdResponsiveInput')
+    set options(value: ResponsiveInputOptions) {
+        this._disabled = value.disabled;
+        this.clrGridRow = !this.disabled;
+    }
+
+    public get disabled(): boolean {
+        return this._disabled;
+    }
+
+    private _disabled = false;
+
     @HostBinding('class.clr-row') public clrGridRow = true;
+
     constructor(private el: ElementRef<HTMLElement>) {}
 
     ngAfterViewChecked(): void {
+        if (this._disabled) {
```

Comment:
Why not use one `if` statement here like:
`if (this._disabled || this.elementWidth !== 0)`

Reply:
I think it's simpler to see multiple early returns

It's clearer in which cases the function does nothing. Otherwise you may need a gigantic comment explaining your boolean expression